engine/crates/runtime/src/fetch.rs

```diff
@@ -60,6 +61,38 @@ impl Fetcher {
             inner: Arc::new(fetcher),
         }
     }
+
+    #[allow(unused_variables)]
+    pub async fn post<SleepFut>(
+        &self,
+        request: &FetchRequest<'_>,
+        sleep: impl Fn(Duration) -> SleepFut + Send + Sync,
```

Comment:
This argument and the `#[allow(unused_variables)]` are remainders of playing with backoff, please remove if not necessary anymore.